src/rust/graph-generator-lib/src/aws.rs

```diff
@@ -0,0 +1,221 @@
+use crate::serialization::SubgraphSerializer;
```

Comment:
This file contains code that is specific to running Grapl in production (on aws).